example/main.go

```diff
@@ -85,6 +85,12 @@ echo 'foo' |
 	cmd.Flags().Float64Var(&baz, "idk", 0.0, "I don't know")
 	cmd.Flags().BoolP("async", "a", false, "Run async")
 	cmd.Flags().BoolVarP(&eerr, "error", "e", false, "Makes the program exit with error")
+	cmd.Flags().String("format", "table", `Pretty-print the output using a Go template or one of the following special values
+'table':            Print output in table format with column headers (default)
+'table TEMPLATE':   Print output in table format using the given Go template
+'json':             Print in JSON format
+'TEMPLATE':         Print output using the given Go template.
```

Comment:
The difference between table template and template is not clear

Even by reading the documentation link ypu provided

Reply:
this is just an example to test fang, its not real :)